src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -4333,11 +4331,13 @@ test_sample_commands (void)
    test_sample_command (test_example_53, 53, db, collection, false);
    test_sample_command (test_example_54, 54, db, collection, true);
    test_sample_command (test_example_55, 55, db, collection, false);
+   test_sample_command (test_example_56, 56, db, collection, true);
    test_sample_command (test_example_57, 57, db, collection, false);
    test_sample_command (test_example_58, 58, db, collection, false);
-   test_sample_command (test_example_56, 56, db, collection, true);
-   test_sample_command (test_example_59, 59, db, collection, true);
-   test_sample_command (test_example_60, 60, db, collection, true);
+   if (test_framework_skip_if_not_replset ()) {
```

Comment:
It might be more consistent to move this check into the sample function(s) as done [here](https://github.com/mongodb/mongo-c-driver/blob/0e54f99dada9db1196f82a7ef868f58ca8b0f022/src/libmongoc/tests/test-mongoc-sample-commands.c#L3022) for `test_sample_causal_consistency()`. Using `test_framework_skip_if_no_txns()` instead of `test_framework_skip_if_not_replset()` might be preferable for similar reasons.